src/OrchardCore.Modules/OrchardCore.Admin/Views/AdminDashboard.cshtml

```diff
@@ -1,4 +1,4 @@
 <div class="alert alert-success">
     <h4>@T["Welcome to Orchard"]</h4>
-    <p>@T["Feel free to browse the menu on the left and discover all its possibilities."]</p>
+    <p>@T["Feel free to browse the menu and discover all its possibilities."]</p>
```

Comment:
Maybe it should just be a different translation for rtl languages ;)

Reply:
May be, but I remove it to avoid the confusing